### PR TITLE
Bump in edifice: ign-rendering5

### PR DESCRIPTION
**Pull request by chapulina**
Bump `ign-rendering` and all its dependencies.

See https://github.com/ignition-tooling/release-tools/issues/331

### CODE REVIEW

**Comment by chapulina** [comment]
Build using https://github.com/ignition-tooling/release-tools/pull/332

[Build Status](https://build.osrfoundation.org/job/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/4376/)

**Comment by chapulina** [comment]
@osrf-jenkins run tests pretty please

File: README.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ for dependency installation instructions for each supported operating system.
     ```
 
     ```
-    sudo apt-get -y install libignition-cmake2-dev libignition-common3-dev libignition-math6-eigen3-dev libignition-plugin-dev libignition-physics3-dev libignition-rendering3-dev libignition-tools-dev libignition-transport9-dev libignition-gui4-dev libignition-msgs6-dev libsdformat10-dev
+    sudo apt-get -y install libignition-cmake2-dev libignition-common3-dev libignition-math6-eigen3-dev libignition-plugin-dev libignition-physics3-dev libignition-rendering3-dev libignition-tools-dev libignition-transport9-dev libignition-gui5-dev libignition-msgs6-dev libsdformat10-dev
```
Comment (j-rivero) [inline README.md:167]:
```suggestion
    sudo apt-get -y install libignition-cmake2-dev libignition-common3-dev libignition-math6-eigen3-dev libignition-plugin-dev libignition-physics3-dev libignition-sensors5-dev libignition-rendering5-dev libignition-tools-dev libignition-transport9-dev libignition-gui5-dev libignition-msgs6-dev libsdformat10-dev
```

**Review by j-rivero — APPROVED** [review]
Minor change in documentation proposed. Loos good.

File: README.md (hunk 1 above)
Comment (chapulina) [reply README.md]: Ouch, the script missed these. Thanks!

**Pull request merged**